src/hypofuzz/hypofuzz.py

```diff
@@ -477,7 +502,7 @@ def has_found_failure(self) -> bool:
 def fuzz_several(*targets_: FuzzProcess, random_seed: Optional[int] = None) -> None:
     """Take N fuzz targets and run them all."""
     random = Random(random_seed)
-    targets = SortedKeyList(targets_, lambda t: t.since_new_cov)
+    targets = SortedKeyList(targets_, lambda t: t.since_new_branch)
 
     # Loop forever: at each timestep, we choose a target using an epsilon-greedy
     # strategy for simplicity (TODO: improve this later) and run it once.
```

Comment:
🗒️ noting for later: we should flush a report when we switch targets (but avoid writing exact-duplicate reports)